tycho-plugins/repository-utils/pom.xml

```diff
@@ -71,6 +71,11 @@
 			<artifactId>git-commit-id-plugin</artifactId>
 			<version>2.1.13</version>
 		</dependency>
+		<dependency>
+		    <groupId>org.tukaani</groupId>
```

Comment:
approved to use in eclipse in https://dev.eclipse.org/ipzilla/show_bug.cgi?id=9941 and others so this is not an issue.